### DISCUSSION TITLE
Give me StartStruck 🌟and i give you back

### DISCUSSION THREAD

**Discussion by d4tdev** [Schweinepriester/github-profile-achievements#461] [Q&A]
https://github.com/d4rtj
Please comment your link, i'll give you soon

**Comment by class-undefined** [comment]
i have done
https://github.com/class-undefined/class-undefined

**Reply by d4tdev** [reply]
Done guy @class-undefined

**Reply by whoami-anoint** [reply]
Okay

**Comment by kensoz** [comment]


**Reply by d4tdev** [reply]
done guy @kensoz

**Comment by ChrisHagedorn** [comment]
Gotta do what you gotta do. Could you toss me GALAXY BRAIN?

https://github.com/ChrisHagedorn

**Reply by d4tdev** [reply]
you need 2 accept answer to get Galaxy Brain

you can answer question in here (https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions?discussions_q=is%3Aunanswered)
if this answer is true, he'll give you marked answer

**Comment by haithamaouati** [comment]
If my answer helped you, you could mark it as answered. Please.

**Comment by 1RB** [comment]
### please star mine aswell 

* 🌟https://github.com/1RB/Authworria🌟

**Reply by d4tdev** [reply]
done guy @1RB

**Comment by MuriloBarranco** [comment]
https://github.com/MuriloBarranco/ignite-challenge-01 
🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟🌟

**Comment by felizardo27** [comment]
https://github.com/felizardo27/pokedex-RN

Please help me
